src/components/layout/footer/bottom-social-wrapper.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import SocialWrapperComponent  from './social-wrapper'
+import { Show } from 'components/containers'
+
+const BottomSocialWrapper = (type) => {
+
+    return (
+        <>
+            <Show.NonEU>
+                <SocialWrapperComponent is_career_page={type === 'careers'} />
+            </Show.NonEU>
+
+            <Show.Eu>
+                <Show.Mobile>
+                    <SocialWrapperComponent is_career_page={type === 'careers'} />
+                </Show.Mobile>
+            </Show.Eu>
```

Comment:
Seems like it should show the `SocialWrapperComponent` in nearly all conditions (NonEu, Eu, Mobile). And the props is the same which seems kinda sus 🤔  What happens if we remove all the `show` component does this still work? If not we can omit this wrapper and just use the `SocialWrapperComponent` directly

Reply:
I think the diff is the position of the `social component` 

Non EU :
- Desktop : shows socialWrapper on top of the footer (rendered in `logo.js`)
- Mobile : shows social wrapper on bottom of the footer (rendered in `bottom-social-wrapper.js`)

EU:
- Desktop & Mobile : shows social wrapper on bottom of the footer (rendered in `bottom-social-wrapper.js`)


as of the career props  it will check if it's from deriv career it will redirect the social to career page ( controlled on `social-wrapper.js`)